src/transform/filter/filterByIsbnIdentifier.js

```diff
@@ -4,23 +4,30 @@ import ConversionError from '../convert/conversionError.js';
  * Filter filtering items without isbn identifiers.
  * @returns Object containing filter and its name
  */
-export function filterByIsbnIdentifier(filterConfig) {
-  const reversed = filterConfig?.filterByIsbnIdentifier?.reverse;
-
-  return {
-    filter,
-    name: 'filterByIsbnIdentifier'
-  };
+export function filterByIsbnIdentifier({active = true, reverse = false}) {
+  if (active) {
+    return {
+      filter,
+      name: 'filterByIsbnIdentifier'
+    };
+  }
+  return false;
 
   function filter({getFieldValues}, debugInfo = {}) {
+    // <kk:field schema="dc" element="identifier" qualifier="isbn" language="none" value="978-xxx-xxx-xxx-8" />
     const isbnIdentifier = getFieldValues('dc.identifier.isbn') || [];
+    // <kk:field schema="dc" element="identifier" qualifier="urn" language="en" value="URN:978-xxx-xxx-xxx-8" />
+    const isbnUrnIdentifier = getFieldValues('dc.identifier.urn').filter(field => field.match(/URN:978-/ui) || field.match(/URN:ISBN:978-/ui)) || [];
```

Comment:
URN-isbn validity needs to be approved by metadata expert before we can implement it.